static/app/components/feedback/widget/useFeedbackWidget.tsx

```diff
@@ -18,7 +21,12 @@ export default function useFeedbackWidget({
   messagePlaceholder,
 }: Props) {
   const config = useLegacyStore(ConfigStore);
-  const feedback = Sentry.getFeedback();
+  const {state} = useAsyncSDKIntegrationStore();
+
+  // TODO(ryan953): remove the fallback `?? Sentry.getFeedback()` after
+  // getsentry is calling `store.add(feedback);`
+  const feedback =
+    (state.Feedback as FeedbackIntegration | undefined) ?? Sentry.getFeedback();
```

Comment:
the `state.Feedback` here must sync up with the setState call in getsentry. Very important stuff!

`Feedback` in this case is also the proper name of the integration, so that's consistent too.